Add gifs to docs and readme (#4)
Tauffer-Consulting/domino: docs/source/pages/domino_components_gui.rst

        <br>
        Edit Pieces by changing their input. Outputs from upstream Pieces are automatically available as inputs for downstream Pieces. Pieces can pass forward any type of data, from simple strings to heavy files, all automatically handled by Domino shared storage system.
        <br>
        <img src="../_static/media/5_edit_pieces.gif" style="margin-top: 20px;">
    </details>

    <br>
    
    <details>
        <summary>
            <strong>Edit and schedule Workflows</strong>
        </summary>
        <br>

>         Schedule Workflows to run periodically, at a specific date/time, or trigger them manually, and select the the best storage for you.

Reviewer: ```suggestion
        Schedule Workflows to run periodically, at a specific date/time, or trigger them manually, and select the the shared storage service.
```
